valgrind.supp

```diff
@@ -0,0 +1,965 @@
+   fun:_ZNK5boost3log11v2_mt_posix9binder1stINS1_10to_log_funINS1_11expressions3tag8smessageEEERNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENS9_9allocatorIcEEEEEclINS9_12basic_stringIcSB_SD_EEEEvRKT_
+   fun:_ZNK5boost3log11v2_mt_posix3aux12singular_refINSt3__112basic_stringIcNS4_11char_traitsIcEENS4_9allocatorIcEEEENS1_11expressions3tag8smessageEE13apply_visitorINS1_9binder1stINS1_10to_log_funISD_EERNS1_24basic_formatting_ostreamIcS7_S9_EEEEEENT_11result_typeESN_
+   fun:_ZN5boost3log11v2_mt_posixlsIcNSt3__111char_traitsIcEENS3_9allocatorIcEENS3_12basic_stringIcS5_S7_EENS1_11expressions3tag8smessageEEERNS1_24basic_formatting_ostreamIT_T0_T1_EESI_RKNS1_9value_refIT2_T3_EE
+   fun:_ZNK5boost5proto6detail18default_shift_leftINS_7phoenix12meta_grammarEE4implIRNS3_5actorINS0_7exprns_10basic_exprINS0_6tagns_3tag10shift_leftENS0_7argsns_5list2INS7_INS9_ISC_NSE_INS7_INS_3log11v2_mt_posix11expressions3aux25attribute_output_terminalINS7_INS9_ISC_NSE_INS7_INSJ_INS7_INS9_ISC_NSE_INS7_INSJ_INS7_INS9_ISC_NSE_INS7_INS9_ISC_NSE_INS7_INS9_ISC_NSE_INS7_INSJ_INS7_INS9_INSB_8terminalENSD_4termINS3_8argumentILi2EEEEELl0EEEEENS_10posix_time5ptimeENSG_16fallback_to_noneENSG_3aux14light_functionIFvRNSG_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENSX_9allocatorIcEEEERKSS_EEEEEEENS7_INS9_ISK_NSL_IA7_cEELl0EEEEEEELl2EEEEENS7_INS9_ISK_NSL_IiEELl0EEEEEEELl2EEEEENS7_INS9_ISK_NSL_IA3_cEELl0EEEEEEELl2EEEEEN2ml4core7CLogger6ELevelEST_NSG_10to_log_funIvEEEEEENS7_INS9_ISK_NSL_IcEELl0EEEEEEELl2EEEEENSX_12basic_stringIcSZ_S11_EEST_S1Z_EEEES24_EELl2EEEEEiST_S1Z_EEEES24_EELl2EEEEENSH_15attribute_actorIS29_ST_NSH_3tag8smessageES7_EEEELl2EEEEERNS3_7vector3IPKS2Q_RKNSG_11record_viewERNSI_10stream_refIS12_EEEERKNS3_15default_actionsEEclES2R_S32_S35_
+   fun:_ZNK5boost7phoenix9evaluator4implIRNS0_5actorINS_5proto7exprns_10basic_exprINS4_6tagns_3tag10shift_leftENS4_7argsns_5list2INS3_INS6_IS9_NSB_INS3_INS_3log11v2_mt_posix11expressions3aux25attribute_output_terminalINS3_INS6_IS9_NSB_INS3_INSG_INS3_INS6_IS9_NSB_INS3_INSG_INS3_INS6_IS9_NSB_INS3_INS6_IS9_NSB_INS3_INS6_IS9_NSB_INS3_INSG_INS3_INS6_INS8_8terminalENSA_4termINS0_8argumentILi2EEEEELl0EEEEENS_10posix_time5ptimeENSD_16fallback_to_noneENSD_3aux14light_functionIFvRNSD_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENSU_9allocatorIcEEEERKSP_EEEEEEENS3_INS6_ISH_NSI_IA7_cEELl0EEEEEEELl2EEEEENS3_INS6_ISH_NSI_IiEELl0EEEEEEELl2EEEEENS3_INS6_ISH_NSI_IA3_cEELl0EEEEEEELl2EEEEEN2ml4core7CLogger6ELevelESQ_NSD_10to_log_funIvEEEEEENS3_INS6_ISH_NSI_IcEELl0EEEEEEELl2EEEEENSU_12basic_stringIcSW_SY_EESQ_S1W_EEEES21_EELl2EEEEEiSQ_S1W_EEEES21_EELl2EEEEENSE_15attribute_actorIS26_SQ_NSE_3tag8smessageES3_EEEELl2EEEEERKNS0_7vector2IRNS0_7vector3IPKS2N_RKNSD_11record_viewERNSF_10stream_refISZ_EEEERKNS0_15default_actionsEEENS4_6envns_9empty_envEEclES2O_S36_RKS38_
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Value8
+   fun:_platform_memmove$VARIANT$Haswell
```

Comment:
This is a machine-specific function that's been chosen based on the machine the suppression file was generated on.  It's probably best to replace it with `...` (see https://wiki.wxwidgets.org/Valgrind_Suppression_File_Howto#Wildcards for syntax).